file_formats/fip.md

```diff
@@ -20,46 +20,72 @@ For example:
 
 ```plaintext
 📦 fib
-┣ FIP_DataG_2024-06-05T08_25_33.csv
-┣ FIP_DataIso_2024-06-05T08_25_33.csv
-┣ FIP_DataR_2024-06-05T08_25_33.csv
-┣ FIP_RawG_2024-06-05T08_25_33.bin
-┣ FIP_RawIso_2024-06-05T08_25_33.bin
-┣ FIP_RawR_2024-06-05T08_25_33.bin
-┣ FIP_ROIsG-Iso_2024-06-05T08_25_33.csv
-┗ FIP_ROIsR_2024-06-05T08_25_33.csv
+┣📂 SoftwareEvents
```

Comment:
After a brief friday discussion and the decision to move a `regions.json` to the root of the data, I think SoftwareEvents should remain undocumented since they will not be needed for any sort of explicit data qc. If we later decide they should, we can always add them in a backwards compatible way.